datumaro/components/dataset_filter.py

```diff
@@ -64,6 +64,8 @@ def encode_annotation_base(cls, annotation):
         ET.SubElement(ann_elem, "type").text = str(annotation.type.name)
 
         for k, v in annotation.attributes.items():
+            if k.isdigit():
+                k = "_" + k
```

Comment:
So, you decided to add _ prefix for export? Do you extract _ in case of import?

Reply:
Yes. As you suggested, I'm using that way with _ prefix. The sub labels in task chains are classified as action in Datumaro. And these _ prefix has been using in action elements with digit. But these action elements are filtered during import. This is the code that is being filtered during import. I checked both export and import, and compared the results between digit case and non-digit case and I checked they had the same results.